### DISCUSSION TITLE
1.4.1-beta

### DISCUSSION THREAD

**Discussion by ilyhalight** [ilyhalight/voice-over-translation#279] [Обновления]
## Список изменений
- Локализация была разделена на отдельные файлы и перенесена на Github

## What's Changed (autogenerated)
* Add localization provider by @MrSoczekXD in https://github.com/ilyhalight/voice-over-translation/pull/275

**Full Changelog**: https://github.com/ilyhalight/voice-over-translation/compare/1.4.0.3...1.4.1-beta

<hr /><em>This discussion was created from the release <a href='https://github.com/ilyhalight/voice-over-translation/releases/tag/1.4.1-beta'>1.4.1-beta</a>.</em>

**Comment by NullVerdict** [comment]
У кого тоже при первом инжекте скрипта не появляется меню, а после перезагрузки оно появляется?

**Comment by NullVerdict** [comment]
В вк почему-то Английский а на ютубе Русский
![изображение](https://github.com/ilyhalight/voice-over-translation/assets/24498484/ae8433a8-e12d-419b-a0b5-908447522f1d)

**Reply by xirthas** [reply]
@SashaXser у тебя точно 1.4.1-beta?
Потому что и
>У кого тоже при первом инжекте скрипта не появляется меню, а после перезагрузки оно появляется?

вроде как пофикшено, и
>В вк почему-то Английский а на ютубе Русский

в вк русский язык нормально работает (Проверял в Chrome и Firefox)

![image](https://github.com/ilyhalight/voice-over-translation/assets/39883614/23f90314-9a0a-4b82-834d-6548a09166e6)

**Reply by ilyhalight** [reply]
> в вк русский язык нормально работает (Проверял в Chrome и Firefox)

у меня тоже норм работает

**Comment by FiLinX** [comment]
млин, у меня опять походу в ютубных эмбедах отвалилось((
сколько не перезагружаю - а кнопку даже нащупать-увидеть немогу(

притом отключив 1.4.1-beta в темпермонке, и включив там оставшуюся стоять отдельно 1.1.0 - то кнопка на эмбедах появляется и перевод включается.. (правда в настройки по троеточию не заходит уже там, но это думаю из-за куков от 1.4.1-beta)

пысы:   откатился до 1.4.0.3 - и всё прекрасно работает на эмбедах)
а подскажите плиз - что в этом 1.4.0.3 скрипте можно отредактировать, чтобы он так же остался у меня в темпермонке запасным?  ну чтоб следущая версия не обновляла этот.

**Reply by ilyhalight** [reply]
VOT - Отредактировать - Нстройки - Обновления - Проверять наличие обновлений (нужно снять галочку) - Сохранить

**Reply by ilyhalight** [reply]
На счет эмбедов, то странно, у меня в них кнопка есть

**Comment by FiLinX** [comment]
> VOT - Отредактировать - Нстройки - Обновления - Проверять наличие обновлений (нужно снять галочку) - Сохранить

так не, я не про автообновления, у меня уже давно ничё авто не обновляется
мне хотелось чтобы при клике здесь на свежую vot.user.js - НЕ предлагало обновить ту 1.4.0.3

а с эмбедами я ещё потом попробую с доскональной чисткой-сбросом настроек скрипта-темпермонки-куков и данных сайтов в браузере

**Reply by ilyhalight** [reply]
Можете поменять name и namespace, после этого не должно предлагать обноволения самого скрипта. Можно просто добавить 1.4.0.3, backup или что-то в таком роде

![image](https://github.com/ilyhalight/voice-over-translation/assets/62353659/540817fe-f9fb-4ee2-924a-c95f39045ca0)
![image](https://github.com/ilyhalight/voice-over-translation/assets/62353659/f35bbe5b-cca5-49b3-9a40-b8174ca516be)
